src/views/Home/components/Question/questionData.js

```diff
@@ -34,14 +34,19 @@ export async function getQuestions() {
   
   await getUserData();
   let backendQuestions = await wellness.getQuestion();
-
-  let phoneNumber = `(${user.MobilePhone.substring(0, 3)}) ${user.MobilePhone.substring(
-    3,
-    6,
-  )}-${user.MobilePhone.substring(6)}`;
-
-  //comment below disables the eslint warning so we can execute dynamic string parsing without warnings
   
+  let phoneNumber;
+  
+  if(user.MobilePhone === undefined){
+     phoneNumber = "your phone number";
```

Comment:
so this phoneNumber varible is eaither going to display a persons phone number or " your phone number". This is like this for the purpose of clarity when the people in the database are writing the questions. They are only going to specify the variable and then the rest is on us. so if they right "we will contact you at {phone number}" it will show up as "we will contact you at 781 526 5849" or " we will contact you at your phone number"